src/components/Button/Button.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+import { BaseColor } from "../../config/color";
```

Comment:
please configure your linter to leave a blank line after import statements, functions and before return statements